### PR TITLE
Using pymssql

### PR DESCRIPTION
**Pull request by megansam5**
solves issue #27 
changes all instances of pyodbc in data-transfer pipeline to pymssql

### CODE REVIEW

File: data-transfer-pipeline/clean.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 from datetime import datetime, timedelta
 from os import environ as ENV
 
-import pyodbc
 from dotenv import load_dotenv
 
 from extract import create_connection
```
Comment (CalebMathers) [inline data-transfer-pipeline/clean.py:7]: Lovely import, its giving DRY

File: data-transfer-pipeline/README.md
Comment (CalebMathers) [inline data-transfer-pipeline/README.md:1]: Solid instructions

File: data-transfer-pipeline/Dockerfile [hunk 2]
```diff
@@ -2,8 +2,21 @@ FROM python:latest
...
+    gcc \
+    g++ \
+    make \
+    curl \
+    && apt-get clean
+
+ENV CFLAGS="-I/usr/include/openssl"
+ENV LDFLAGS="-L/usr/lib/x86_64-linux-gnu"
+ENV CPPFLAGS="-I/usr/include/openssl"
+
```
Comment (CalebMathers) [inline data-transfer-pipeline/Dockerfile:16]: 200 IQ

File: data-transfer-pipeline/extract.py [hunk 3]
```diff
@@ -3,43 +3,41 @@
 from os import environ as ENV
 
 import pandas as pd
-import pyodbc
+import pymssql
```
Comment (CalebMathers) [inline data-transfer-pipeline/extract.py:6]: We don't like you anymore pyodbc, pymssql is our friend now

File: data-transfer-pipeline/extract.py [hunk 4]
```diff
@@ -3,43 +3,41 @@
...
 
-    cutoff_time = datetime.now() - timedelta(days=1)
     conn = create_connection()
+    with conn.cursor() as cursor:
+        cursor.execute(query, (cutoff_time,))
+        res = cursor.fetchall()
 
-    df = pd.read_sql(query, conn, params=[cutoff_time])
     conn.close()
+    df = pd.DataFrame(res)
```
Comment (CalebMathers) [inline data-transfer-pipeline/extract.py:38]: Could use a slightly nicer variable name for this

File: data-transfer-pipeline/requirements.txt
[the remark this answers is not in the document]
Comment (megansam5) [reply data-transfer-pipeline/requirements.txt:1]: you are a genius

File: data-transfer-pipeline/clean.py (hunk 1 above)
Comment (megansam5) [reply data-transfer-pipeline/clean.py:7]: ok rude

Comment (megansam5) [reply data-transfer-pipeline/clean.py:7]: sorry not rude

File: data-transfer-pipeline/README.md
Comment (megansam5) [reply data-transfer-pipeline/README.md:1]: thanks

File: data-transfer-pipeline/Dockerfile (hunk 2 above)
Comment (megansam5) [reply data-transfer-pipeline/Dockerfile:16]: i know

File: data-transfer-pipeline/extract.py (hunk 3 above)
Comment (megansam5) [reply data-transfer-pipeline/extract.py:6]: yes

File: data-transfer-pipeline/extract.py (hunk 4 above)
Comment (megansam5) [reply data-transfer-pipeline/extract.py:38]: damn the QA is QAing

**Pull request merged**